### PR TITLE
Fix eQ URL in pipelines to conform to new style without path

### PR DESCRIPTION
**Pull request by jcox-dev**
# Motivation and Context

Follows changes introduced in [respondent-home-ui#35](https://github.com/ONSdigital/respondent-home-ui/pull/35).

# What has changed
Remove path from EQ_URL.

### CODE REVIEW

File: pipelines/deployment.yml [hunk 1]
```diff
@@ -1129,7 +1129,7 @@ jobs:
       environment_variables:
         <<: *ci_security_user
         <<: *ci_service_endpoints_for_python
-        EQ_URL: 'https://eq-test/session?token='
```
Comment (liamtoozer) [inline pipelines/deployment.yml]: Is there going to be a change in frontstage for this URL bit as well?

Comment (jcox-dev) [reply pipelines/deployment.yml]: @liamtoozer no, my mistake!

**Review by liamtoozer — APPROVED** [review]

**Comment by benjefferies** [comment]
Didn't see this one @jcox-dev 🤦‍♂️

**Comment by jcox-dev** [comment]
Fixed in #55

**Pull request closed without merging**